packages/vscode-ui5-language-assistant/test/runIntegrationTests.ts

```diff
@@ -1,32 +1,24 @@
 import { resolve, dirname } from "path";
 import { runTests } from "vscode-test";
+import globby from "globby";
 
 async function main(): Promise<void> {
   try {
     const pkgJsonPath = require.resolve(
       "vscode-ui5-language-assistant/package.json"
     );
     const rootPkgFolder = dirname(pkgJsonPath);
+    const testPkgFolder = resolve(rootPkgFolder, "lib", "test", "suite");
 
     const extensionDevelopmentPath = resolve(rootPkgFolder);
-    const extensionTestsPath = resolve(
-      rootPkgFolder,
-      "lib",
-      "test",
-      "suite",
-      "index"
-    );
-    const extensionTestWorkspace = resolve(
-      rootPkgFolder,
-      "test",
-      "testFixture"
-    );
 
-    await runTests({
-      extensionDevelopmentPath,
-      extensionTestsPath,
-      launchArgs: [extensionTestWorkspace, "--disable-extensions"],
-    });
+    const scenarioPaths = await globby(`${testPkgFolder}/**/index.js`);
+    for (const path of scenarioPaths) {
+      await runTests({
```

Comment:
Please document the usage of `for of` loop to ensure `runTests` are performed in **sequence**